wbooks/src/redux/book/actions.ts

```diff
@@ -1,15 +1,22 @@
 import { createTypes, completeTypes } from 'redux-recompose';
-import { getBooks } from '@services/BookService';
+import { getBooks, getFilteredBooks } from '@services/BookService';
 
-export const actions = createTypes(completeTypes({ primaryActions: ['GET_BOOKS'] }), '@@BOOK');
-
-const TARGET = 'books';
+export const actions = createTypes(
+  completeTypes({ primaryActions: ['GET_BOOKS', 'GET_FILTERED_BOOKS'] }),
+  '@@BOOK'
+);
 
 const actionCreators = {
   getBooks: () => ({
     type: actions.GET_BOOKS,
-    target: TARGET,
+    target: 'books',
     service: getBooks
+  }),
+  getFilteredBooks: (query: string) => ({
+    type: actions.GET_FILTERED_BOOKS,
+    target: 'filteredBooks',
+    service: getFilteredBooks,
+    payload: query
```

Comment:
Just as a comment: In the next card you must use `useMemo` and do the filter in the Filter screen, not in the service/backend

Reply:
Okay, so in the next card I need to delete the `getFilteredBooks` `service`, `reducer` and `action`, right?